src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_ouro.cpp

```diff
@@ -17,11 +17,75 @@ EndScriptData */
 
 enum Spells
 {
+    // Ouro
     SPELL_SWEEP                 = 26103,
     SPELL_SANDBLAST             = 26102,
     SPELL_GROUND_RUPTURE        = 26100,
     SPELL_BIRTH                 = 26262, // The Birth Animation
-    SPELL_DIRTMOUND_PASSIVE     = 26092
+    SPELL_DIRTMOUND_PASSIVE     = 26092,
+
+    // Ouro Spawner
+    SPELL_SUMMON_OURO           = 26061 // Summons Ouro, used by the Ouro Spawner
+};
+
+enum Events
+{
+    EVENT_SPELL_SWEEP = 1,
+    EVENT_SPELL_SANDBLAST = 2,
+    EVENT_SUBMERGE = 3,
+    EVENT_BACK = 4,
+    EVENT_CHARGE_TARGET = 5,
+    EVENT_SPAWN = 6
+};
+
+class npc_ouro_spawner : public CreatureScript
+{
+public:
+    npc_ouro_spawner() : CreatureScript("npc_ouro_spawner") { }
+
+    CreatureAI* GetAI(Creature* creature) const
```

Comment:
Missing override

Reply:
Okay, done :)

Reply:
Still missing here :P Also the whole tabs got shifted.

Reply:
Nevermind, the tabs didn't get shifted.